lang/wren/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem        1.0
+PortGroup         github 1.0
+PortGroup         makefile 1.0
+
+github.setup      wren-lang wren 0.3.0
+
+categories        lang
+license           MIT
+platforms         darwin
+
+description       Wren is a small, fast, class-based concurrent scripting \
+                  language.
+
+long_description  {*}${description} Think Smalltalk in a Lua-sized package \
+                  with a dash of Erlang and wrapped up in a familiar, modern \
+                  syntax.  Wren is intended for embedding in applications. It \
+                  has no dependencies, a small standard library, and an \
+                  easy-to-use C API. It compiles cleanly as C99, C++98 or \
+                  anything later.
+
+homepage          https://wren.io
+
+maintainers       {gmail.com:herby.gillot @herbygillot} \
+                  openmaintainer
+
+checksums         rmd160  2b0acc5b21184ff5f8bb485def843418e9a7ca3f \
+                  sha256  489ac252ec805bfa7aab07286e2bc8dc6348bdb55857f7afb9640c5919e118ed \
+                  size    1228925
+
+worksrcdir        ${worksrcdir}/projects/make.mac
+
+build.args-append verbose=1
+build.target      wren_shared
+
+if {${os.arch} eq "i386"} {
+    build.args-append config=release_32bit
+} else {
+    build.args-append config=release_64bit
+}
```

Comment:
This looks like you're assuming Intel only. If so, I recommend adding `supported_archs x86_64 i386`. Or if these configurations only deal with bitness and don't care about architecture, then it should be `if {${os.arch} in "i386 ppc"}`.

Reply:
OK changes made